edgecast/waf/custom_rule.go

```diff
@@ -355,3 +355,24 @@ func (svc *WAFService) GetCustomRuleSet(accountNumber string, customRuleID strin
 
 	return parsedResponse, nil
 }
+
+//UpdateCustomRuleSet an access rule that identifies valid or malicious requests via whitelists, accesslists, and blacklists.
```

Comment:
Need to update this comment

Reply:
changed

Reply:
Hmm that description is for a custom rule set in general. I think just `// Updates a custom rule` should be enough

Reply:
i got this from help doc on ticket so more info should be good.